ktor-server/ktor-server-plugins/ktor-server-sse/common/src/io/ktor/server/sse/ServerSSESession.kt

```diff
@@ -130,3 +133,36 @@ public suspend inline fun <reified T : Any> ServerSSESessionWithSerialization.se
 public suspend inline fun <reified T : Any> ServerSSESessionWithSerialization.send(data: T) {
     send(ServerSentEvent(serializer(typeInfo<T>(), data)))
 }
+
+/**
+ * Starts a heartbeat for the ServerSSESession.
+ *
+ * The heartbeat will send the specified [Heartbeat.event] at the specified [Heartbeat.duration] interval
+ * as long as the session is active.
+ *
+ * @param heartbeatConfig a lambda that configures the [Heartbeat] object used for the heartbeat.
+ */
+public fun ServerSSESession.heartbeat(heartbeatConfig: Heartbeat.() -> Unit) {
```

Comment:
Should we add a default value here?

```suggestion
public fun ServerSSESession.heartbeat(heartbeatConfig: Heartbeat.() -> Unit = {}) {
```